src/main/java/org/jfree/chart/block/ColumnArrangement.java

```diff
@@ -308,6 +310,7 @@ protected Size2D arrangeRF(BlockContainer container, Graphics2D g2,
      *
      * @return The size after the arrangement.
      */
+    @SuppressWarnings("index") // https://github.com/kelloggm/checker-framework/issues/152

```

Comment:
I would add "@LengthIs" to the comment, before the URL, to make it easier for readers to know what the warning suppression is about.  The same comment applies to other comments that consist of just a URL:  writing one or two words to explain them is helpful.